doc/source/development/maintaining.rst

```diff
@@ -151,16 +151,37 @@ Here are some things to check when reviewing a pull request.
   for regression fixes and small bug fixes, the next minor milestone otherwise)
 * Changes should comply with our :ref:`policies.version`.
 
+
+.. _maintaining.backporting:
+
 Backporting
 -----------
 
-In the case you want to apply changes to a stable branch from a newer branch then you
-can comment::
+pandas supports point releases (e.g. ``1.4.3``) that aim to:
+
+1. Fix bugs in new features introduced since the first minor version release.
```

Comment:
Maybe with instead of since?

Reply:
Hmm fair point. Maybe `introduced in the first minor version release`

Reply:
Yep sounds good